### PR TITLE
wix-ui-mocha-runner: add readme

### PR DESCRIPTION
**Pull request by alisey**

### CODE REVIEW

**Comment by hepiyellow** [comment]
If you open the branch under `wix-ui` then I can commit my suggested changes, instead of writing about them.

File: packages/wix-ui-mocha-runner/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# wix-ui-mocha-runner
+
+Run unit tests in headless Chromium using Mocha and Puppeteer. The test results are logged to the terminal, and all console output is redirected from the browser to the terminal as well.
```
Comment (hepiyellow) [inline packages/wix-ui-mocha-runner/README.md]: 1. Say how to run it NOT in watch mode:
`npx wix-ui-mocha-runner`
and then what happens with results and console logs.

2. Can you tell about which assertion library is used.
 Tests are written with jest, which has jest assertion library. I guess you some how load jest assertion library that runs with Mocha.

File: packages/wix-ui-mocha-runner/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# wix-ui-mocha-runner
+
```
Comment (hepiyellow) [inline packages/wix-ui-mocha-runner/README.md]: Can you tell a bit about :
- the motivation to run tests in browser. Why it is better than alternatives like jsdom, puppeteer.
- are there any limitations. Can the mocha runner runnin gin browser interfere with the production code? is there sandboxing?
- How it is achieved (bundle) if there is something interesting

File: packages/wix-ui-mocha-runner/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+Run unit tests in headless Chromium using Mocha and Puppeteer. The test results are logged to the terminal, and all console output is redirected from the browser to the terminal as well.
+
+To run tests interactively in watch mode run `npx wix-ui-mocha-runner --watch` in the project root, then open [http//127.0.0.1:7357](http//127.0.0.1:7357). You can click the play icon to the right of the test name or a group of tests to run only a subset of tests. If the test page starts to constantly refresh make sure it's not open in mutliple tabs.
```
Comment (hepiyellow) [inline packages/wix-ui-mocha-runner/README.md]: Please separate to bullets:
- mention hot-reload for those of us who don't know what watch mode it (-:
- Tell us about the right side panel
- clicking component name for focused tests
- refresh issue (maybe put it under TroubleShooting heading)

Comment (hepiyellow) [inline packages/wix-ui-mocha-runner/README.md]: Future plans:
- snapshot (eyes) integration ?
If its not related directly to mocha runner, maybe just mention eyes integration in the motivation section.
- wallaby integration ?

File: packages/wix-ui-mocha-runner/README.md (hunk 2 above)
Comment (alisey) [reply packages/wix-ui-mocha-runner/README.md]: 1. One is not strictly better than the other, there are many different tradeoffs, and it's up to the consumer to choose what they want.
2. Don't understand the question.
3. Webpack, basically.

File: packages/wix-ui-mocha-runner/README.md (hunk 3 above)
Comment (alisey) [reply packages/wix-ui-mocha-runner/README.md]: 1. Done.
2. It's a standard Mocha html reporter, I don't think they even have documentation for it on their website.

Comment (alisey) [reply packages/wix-ui-mocha-runner/README.md]: Maybe it could be made into something more useful / generic, but I don't have any future plans right now.

**Comment by alisey** [comment]
@hepiyellow added more details

**Pull request merged**